SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -176,6 +178,9 @@ chmod 000 %{_sysconfdir}/shadow
 %{_libdir}/libsubid.so
 
 %changelog
+* Wed Sep 20 2023 Kanika Nema <kanikanema@microsoft.com> - 4.9-13
```

Comment:
NIT: 
I would request to take the suggestions from the linter:
At-least the ones that:
`/usr/sbin` -> %{_sbindir}`

Reply:
Sorry, didn't pick up this comment. The builds have completed, and I didn't want to update the spec file and not run a build (even if it is just tp take in lint changes). Thanks for the comment, I will look through the lint errors a bit more carefully next time.